elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideAsyncConfiguration.java

```diff
@@ -98,9 +100,17 @@ public AsyncCleanerService buildAsyncCleanerService(Elide elide, ElideConfigProp
     public AsyncQueryDAO buildAsyncQueryDAO(Elide elide) {
         // Creating a new ElideSettings and Elide object for Async services
         // which will have ISO8601 Dates. Used for DefaultAsyncQueryDAO.
+        String serdePattern = "yyyy-MM-dd'T'HH:mm'Z'";
+        String serdeTimeZone = "UTC";
+        if (elide.getElideSettings().getSerdes().get(Date.class).getClass().getName()
```

Comment:
I think if the serde is not defined for the main elide object, then `getClass().getName()` can throw a Null Pointer Exception.

Reply:
it gives me an EpochDateConverter... doesnt throw npe